dataland-internal-storage/src/test/kotlin/db/migration/utils/MigrateFrameworkMockAndWhenConfiguation.kt

```diff
@@ -21,7 +21,7 @@ fun mockAndWhenConfigurationForFrameworkMigration(
     val mockResultSet = Mockito.mock(ResultSet::class.java)
     Mockito.`when`(mockResultSet.getString("data_id")).thenReturn("data-id")
     Mockito.`when`(mockResultSet.getString("data")).thenReturn(originalDatabaseEntry)
-    Mockito.`when`(mockResultSet.next()).thenReturn(true, false)
+    Mockito.`when`(mockResultSet.next()).thenReturn(true, false, true, false)
```

Comment:
Wofür stehen die hinzugefügten true, false params?

Reply:
Für die Sequenz, in der "next()" entweder true oder false returnen soll.

Warum genau diese Sequenz hier gewählt werden muss, müssten wir die anderen fragen.  Das kann ich so nicht erkennen.

Reply:
Es sagt quasi aus:

"Wenn `next()` aufgerufen wird, dann returne true. Beim nächsten Aufruf dann false. Danach wieder true, danach wieder false.

Reply:
Ich close hier. Ich werde am Montag Florian mal fragen, was der Hintergrund ist.